src/components/icons/acastPodcastsIcon.astro

```diff
@@ -0,0 +1,7 @@
+<svg width="24" height="24" viewBox="0 0 24 24">
+  <g fill="none" stroke="#818181" stroke-width=".2">
```

Comment:
Astropod is a multi theme website and we need to add support to change the icon color.

Here in stroke let's change that color for "currentColor"